raco/backends/myria/myria.py

```diff
@@ -1721,6 +1824,193 @@ def fire(self, op):
         return algebra.UnionAll(children)
 
 
+class FillInJoinPullOrder(rules.Rule):
```

Comment:
I agree it's an awkward design, however exposing it in `DO UNTIL CONVERGENCE` is ugly as well. Ideally this kind of physical level tuning should be done by a cost-based optimizer... On the other hand, it doesn't hurt if we can declare all the joins in a program to pull from some side, it might be a good optimizations too (e.g. saving one hash table when first consuming only one child). So I ended up doing this. I've also thought about not setting this arg in RACO, only doing this manually on JSON. What do you think?